convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -35,7 +35,7 @@
 from convert2rhel import pkghandler, pkgmanager, utils
 from convert2rhel.systeminfo import system_info
 from convert2rhel.toolopts import tool_opts
-from convert2rhel.unit_tests import GetLoggerMocked, is_rpm_based_os
+from convert2rhel.unit_tests import GetLoggerMocked, is_rpm_based_os, run_subprocess_side_effect
```

Comment:
Does this need to be here?  I don't see that it is used inside of this file.

Reply:
Good catch, this was left out when I was trying to remember how to mock the run_subporcess! Already removed.